### ISSUE TITLE
Investigate adding focus state only on keyboard

### ISSUE THREAD

**Issue by danoc** [thumbtack/thumbprint#7]
https://twitter.com/ryanflorence/status/1084108070400098305?s=09

This is currently supported by Chrome. Worth doing?

**Comment by danoc** [comment]
I removed this BTW from the docs a while back to keep the our documentation environment similar to the that of the consumers that use Thumbprint.

**Comment by danoc** [comment]
Probably shouldn't use since it's still experimental:

https://caniuse.com/#feat=mdn-css_selectors_focus-visible
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#Specifications

@tomgenoni — What do you think?

**Comment by tomgenoni** [comment]
Yeah, it were was simple as `@support` and it was a clean progressive enhancement I'd say go for it. Doesn't seem worth the effort right now.

**Comment by danoc** [comment]
Agreed. Closing this. We can revisit once the spec is finalized.

**Comment by danoc** [comment]
Re-opening this so we can track since the topic comes up from time to time and we'd evenutally like to do something about it.
